### ISSUE TITLE
[COMMUNITY] 2024-02 Speak at State of Open Con UK, Engage people in a feedback session to hear their needs/wants

### ISSUE THREAD

**Issue by PaulaPaul** [finos/a11y-theme-builder#780]
## Problem/Concern

@lwnoble and Cody Zuschlag have been accepted to present at the State of Open Con in London, Paula is the track host for the Finance Track and will be introducing them, with a plug for getting involved :) 

Paula is reaching out to the organizers to inquire about a collaboration space where we can invite people to bring their feedback and ideas for the project.

* [State of Open Con](https://stateofopencon.com/)

## Proposed Solution

@lwnoble has a vision for the presentation that supports the roadmap for 2024 - our big theme for 2024 is enabling adoption by making it easier for people to create new design systems and end to end accessibility with Theme Builder, but also address the 'BYOC' (Bring your own code) and 'BYOD' (bring your own design system) cases to allow for strategic/incremental adoption of Theme Builder-

This presentation will give us the groundwork for the 2024 roadmap and the overall theme of enabling adoption -

We can track comments here and post comments about the event, including the attendance at the presentation and follow on discussions at the conference.  (this can help us track our community outreach for 2024 in github)
After the conference concludes and we've captured feedback in comments here, we can close this issue.

**Comment by PaulaPaul** [comment]
@lwnoble can you post a read only link to the presentation here for reference? Thank you!!!!

**Comment by PaulaPaul** [comment]
@lwnoble and Cody Zuschlag spoke at the [State of Open Con24](https://stateofopencon.com/) conference - it was a great event and we also joined a FINOS social afterwards where we all spoke to attending companies who were interested in learning more and potentially adopting Theme Builder. The common thread for pain points and needs aligns with this year's roadmap to come up with an approach (and tooling if possible) to address the case where companies already have an existing design system or branding. That approach (and potential tooling) could start with some written materials for how to assess 'where you are'.  This could potentially be another repository for Theme Builder - we have Theme Builder and the SDK, and we could have a separate documentation repo for assessment approaches and tooling, or we could put that documentation under the Theme Builder repository.  

The video from Lise and Cody's talk will be available soon, and I will open a separate issue to create a resource page or other location to publish videos from community events like this.

Leaving this issue open for discussion at our next project call to potentially open an issue related to assessment approach, then we can close this community activity as complete.

**Comment by PaulaPaul** [comment]
If Discover has a public YouTube we could post it there @aaronreed708 can you ask?

**Comment by PaulaPaul** [comment]
We decided to post the link to the video in the Docs section of the repo (on Feb 22's project call). @PaulaPaul will open an issue to do this and once that PR is merged we can close this issue and use this process for any community/conference videos going forward. fyi - @aaronreed708 @lwnoble

**Comment by PaulaPaul** [comment]
Link to the video has been included as working PR for #798 .
Closing this as complete-
